common/common.go

```diff
@@ -551,6 +591,31 @@ func RunBuild(verbose bool, dir, imageName, dockerfile string, buildArgs []strin
 		fmt.Fprintln(os.Stderr)
 		return fmt.Errorf("build cancelled on signal %v", signal)
 	}
+	if containerEngineType != containerEngineTypeDocker || issuePush {
+		fmt.Println("Using Container engine", containerEngineType, "to push as --push id only for docker and --load is for podman")
```

Comment:
Optimise these logs.

Log statement need not be true because we use push even for podman with Build command